src/tap.h

```diff
@@ -32,9 +32,38 @@
 #include <iostream>
 #include <map>
 #include <fstream>
+#include <string>
 
 namespace tap {
 
+// based on http://stackoverflow.com/a/7724536/831180
```

Comment:
Would it be possible to have a test for this... maybe a test written with GoogleTest, testing a part of gtest-tap-listener, generating TAP, with GoogleTest and gtest-tap-listener? 

![2f45e50ff3ea01261f10be92641e728269040849c000e3892913ddf67ade2bbb](https://cloud.githubusercontent.com/assets/304786/7607749/0a1b07f4-f9b8-11e4-8600-7147a5d3e56e.jpg)